### PR TITLE
Health Analyzer Sounds

### PR DESCRIPTION
**Pull request by alsoandanswer**
Title. By the very talented Identity Crisis#2599 on Spritercord.

Currently tied to brain activity.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -21,14 +21,19 @@ BREATH ANALYZER
...
 	var/mode = 1
+	var/last_scan = 0
 
 /obj/item/device/healthanalyzer/attack(mob/living/M, mob/living/user)
-	health_scan_mob(M, user, mode)
-	add_fingerprint(user)
+	if(last_scan <= world.time - 20) //Spam limiter.
+		last_scan = world.time
+		health_scan_mob(M, user, mode)
+		add_fingerprint(user)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/scanners.dm:30]: Don't limit the scanning itself, limit the playsound() instead.

**Review by NonQueueingMatt — DISMISSED** [review]

File: code/game/objects/items/devices/scanners.dm (hunk 1 above)
Comment (SleepyGemmy) [reply code/game/objects/items/devices/scanners.dm:30]: i playtested and it seems fine. prevents yourself from spamming the chat.

Comment (SleepyGemmy) [reply code/game/objects/items/devices/scanners.dm:30]: fair enough!

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**